drag_and_drop_v2/templates/html/drag_and_drop.html

```diff
@@ -1,4 +1,4 @@
 {% load i18n %}
-<section class="themed-xblock xblock--drag-and-drop">
+<div class="themed-xblock xblock--drag-and-drop">
     <i class="fa fa-spin fa-spinner initial-load-spinner"></i>{% trans "Loading drag and drop problem." %}
```

Comment:
The i tag, which is for italic, should not be used for icons. Please go through and change any i tags that are being used for icons.

http://ux.edx.org/design_elements/icons/